netbeans/CheckerFrameworkPlugin/release/checkerstrings.properties

```diff
@@ -0,0 +1,18 @@
+org.checkerframework.checker.nullness.NullnessChecker=Nullness Checker
```

Comment:
Can you add a comment here and state that the order of elements is important, as it is used for ordering the elements in the GUI. (That's my assumption... if the ordering doesn't matter, please alphabetize.)